### ISSUE TITLE
Algolia doesn't work in CP search

### ISSUE THREAD

**Issue by markzegarelli** [statamic/v2-hub#827]
### Steps to reproduce
1. Use Algolia Search driver. Make sure content is in index.
2. Try to use the CP search panel
3. Hold head in confusion (optional).

### Expected behaviour
A list of content should appear.

### Actual behaviour
No results appear

### Server configuration

**Statamic version:**
2.0.11

#### Browser log
Request
```
Request URL:http://support-site-2.dev/cp/search/perform?q=display
Request Method:GET
Status Code:200 OK
Remote Address:127.0.0.1:80
```
Response
```
[]
```
Request is being routed to the built-in search, it looks like.

**Comment by jasonvarga** [comment]
Ah step 3 is optional

**Comment by markzegarelli** [comment]
Sorry, I've updated to reflect that.

**Comment by jasonvarga** [comment]
Thanks, that helps!

**Comment by edalzell** [comment]
I'm on zend and it doesn't work either.

**Comment by jasonvarga** [comment]
It's fixed for 2.1.16
